python/cortex.py

```diff
@@ -102,7 +102,11 @@ def open(self):
                                         on_error=self.on_error,
                                         on_close=self.on_close)
         threadName = "WebsockThread:-{:%Y%m%d%H%M%S}".format(datetime.utcnow())
+        
+        # you might need to use Emotiv self-signed certificate when connect to remote Cortex
+        # sslopt = {'ca_certs': "../docs/rootCA.pem", "cert_reqs": ssl.CERT_REQUIRED}
         sslopt={"cert_reqs": ssl.CERT_NONE}
```

Comment:
We will always recommend 3rd developer using certificate verification to have the best security.